typhos/ui/widgets/positioner.ui

```diff
@@ -47,12 +47,18 @@
    <item alignment="Qt::AlignHCenter">
     <widget class="QFrame" name="motion_frame">
      <property name="sizePolicy">
-      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
+      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
+      <size>
+       <width>16777215</width>
+       <height>16777215</height>
```

Comment:
Was there a reason for this number?  Asking out of curiosity is all.

Reply:
This is the maximum number that designer lets you put in.